app/frontend/react/components/ReviewsContainer.js

```diff
@@ -0,0 +1,50 @@
+  import React, { Component } from 'react';
+  import ReviewTile from './ReviewTile';
+
+  class ReviewsContainer extends Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        reviewsData: []
+      };
+    }
+
+    componentWillMount() {
```

Comment:
Although componentWillMount works, I would suggest using componentDidMount instead because you will typically run into fewer problems if the component is mounted before you change the state.